jobs/tools/ICON_to_point.py

```diff
@@ -0,0 +1,362 @@
+import numpy as np
+import xarray as xr
+from sklearn.neighbors import BallTree
+from scipy import argmin
+import argparse
+
+# TODO: re-use the interpolator & sampler when the function is called with a list of files.
+#       I've tested some options, but I fail to achieve better than linear scaling of the
+#       performance, whereas the second (etc.) calls to the function can entirely skip
+#       recomputing the interpolation weights etc. Probably just a limitation of this method
+#       and a limitation of passing around and merging/concatenating xarray data.
```

Comment:
Is this note needed? If yes, please add your name/initials.

Reply:
OK!